src/lib/pay-request/services/admin_users/types.ts

```diff
@@ -76,6 +76,11 @@ export class User {
     this.features = userData.features ? userData.features.split(',').map((feature: string) => feature.trim()) : []
     this.second_factor = userData.second_factor
     this.provisional_otp_key = userData.provisional_otp_key || ''
+    this.login_counter = userData.login_counter
+    this.provisional_otp_key_created_at = userData.provisional_otp_key_created_at
+    this.last_logged_in_at = userData.last_logged_in_at
```

Comment:
Fields are [camel case in selfservice](https://github.com/alphagov/pay-selfservice/blob/master/src/models/GatewayAccount.class.js#L34) 😞